context/src/main/java/io/micronaut/scheduling/cron/CronExpression.java

```diff
@@ -290,6 +290,15 @@ private static void checkIfDateTimeBarrierIsReached(ZonedDateTime nextTime, Zone
         }
     }
 
+    /**
+     * Returns String expression.
+     *
+     * @return The underlying cron expression as string.
```

Comment:
Please add `@since 3.1.0` since this is a new method

Reply:
done, @graemerocher If it is good to merge, can u please add `hacktoberfest-accepted` label to this PR